default.nix

```diff
@@ -41,6 +41,7 @@ let
 in
 rec {
 
+  agdaWithStdLibMeta = agdaWithPkgs deps;
```

Comment:
I think it's better to make this dependency explicit,  and it's better to cache this one, since the other one (with ledger) is used externally and doesn't have any dependencies in this file , so the dependency to the cache is less obvious